data/json/npcs/random_encounters/refugee_caravans.json

```diff
@@ -0,0 +1,296 @@
+[
+  {
+    "type": "npc",
+    "id": "NC_caravan_refugee_A",
+    "//": "Random NPC, can direct you to the free merchants and nearest FEMA camp.",
+    "name_suffix": "refugee",
+    "class": "NC_SCAVENGER",
+    "attitude": 0,
+    "mission": 7,
+    "chat": "TALK_CARAVAN_REFUGEE_A1",
+    "faction": "wasteland_scavengers"
+  },
+  {
+    "type": "npc",
+    "id": "NC_caravan_refugee_B",
+    "//": "Random NPC, can be recruited away.",
+    "name_suffix": "refugee",
+    "class": "NC_SCAVENGER",
+    "attitude": 0,
+    "mission": 8,
+    "chat": "TALK_CARAVAN_REFUGEE_B1",
+    "faction": "wasteland_scavengers"
+  },
+  {
+    "type": "npc",
+    "id": "NC_caravan_refugee_C",
+    "//": "Random NPC, demands heroin and can accept.",
+    "name_suffix": "refugee",
+    "class": "NC_SCAVENGER",
+    "attitude": 0,
+    "mission": 7,
+    "chat": "TALK_CARAVAN_REFUGEE_C1",
+    "faction": "wasteland_scavengers"
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_CARAVAN_REFUGEE_A1",
+    "dynamic_line": "Please don't hurt us, we're just trying to get to safety.",
+    "speaker_effect": { "effect": { "npc_first_topic": "TALK_CARAVAN_REFUGEE_A2" } },
+    "responses": [
+      {
+        "text": "What if I want to hurt you?  I'm so good at it and I've had so much practice.",
+        "topic": "TALK_CARAVAN_REFUGEE_A1_Hurt"
+      },
+      { "text": "What's your situation here?", "topic": "TALK_CARAVAN_REFUGEE_A1_Job" },
+      { "text": "I don't have time for zombie bait like you.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_CARAVAN_REFUGEE_A2",
+    "dynamic_line": "You again.  Any good news?  Haven't heard any in a long time.",
+    "responses": [
+      { "text": "What's your story?", "topic": "TALK_CARAVAN_REFUGEE_A_Story" },
+      { "text": "No, I don’t have any.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_CARAVAN_REFUGEE_A1_Hurt",
+    "dynamic_line": "I can’t tell if you’re joking or not, please don’t say such things.",
+    "responses": [
+      { "text": "Oh, I wasn’t joking <name_b>.", "effect": "hostile", "topic": "TALK_DONE" },
+      { "text": "What's your situation here?", "topic": "TALK_CARAVAN_REFUGEE_A1_Job" },
+      { "text": "I’m sorry, I should be going.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_CARAVAN_REFUGEE_A1_Job",
+    "dynamic_line": "We're just trying to find a safe place right now.  I think there's a FEMA camp nearby that should have room for us.",
+    "responses": [
+      { "text": "That's it?  What's your story?", "topic": "TALK_CARAVAN_REFUGEE_A_Story" },
+      { "text": "Good luck with that.  I'll be on my way.", "topic": "TALK_DONE" },
+      {
+        "text": "You're never going to make it, you should just end it now.",
```

Comment:
This too.

Reply:
Deleted, this chain is already blocked by the psychopath trait now